com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/HDRenderPipeline.VolumetricCloudsAccumulation.cs

```diff
@@ -166,7 +166,7 @@ VolumetricCloudsParameters_Accumulation PrepareVolumetricCloudsParameters_Accumu
         }
 
         static void TraceVolumetricClouds_Accumulation(CommandBuffer cmd, VolumetricCloudsParameters_Accumulation parameters,
-            RTHandle colorBuffer, RTHandle depthPyramid, TextureHandle motionVectors, TextureHandle volumetricLightingTexture, TextureHandle scatteringFallbackTexture,
+            RTHandle colorBuffer, RTHandle depthPyramid, RTHandle motionVectors, RTHandle volumetricLightingTexture, RTHandle scatteringFallbackTexture,
```

Comment:
Why the change to RTHandle here?

Reply:
it was a mistake, should have been rthandle all long